subsys/dm/dm.c

```diff
@@ -60,6 +60,10 @@ LOG_MODULE_REGISTER(nrf_dm, CONFIG_DM_MODULE_LOG_LEVEL);
 #define DM_THREAD_PRIORITY           K_HIGHEST_APPLICATION_THREAD_PRIO
 #define TIMESLOT_TIMEOUT_STEP_MS     120000
 
+#define DM_TIMESLOT_OVERHEAD_US      400
+#define DM_REFLECTOR_OVERHEAD_US     2000
+#define DM_RANGING_EXTRA_TIMEOUT_US  200
```

Comment:
I guess this one will be removed once we have the fix from nrf_lib_dm library. I could provide you a lib that includes the fix if you want to test.

Reply:
Yes, `DM_RANGING_EXTRA_TIMEOUT_US` will be removed when we have the fix in nrf_lib_dm.